Parse/src/test/java/com/parse/ParseInstallationTest.java

```diff
@@ -184,6 +194,8 @@ public void testUpdateBeforeSave() throws Exception {
     // Make sure we update device info
     assertEquals("android", installation.getString(KEY_DEVICE_TYPE));
     assertEquals(installationId.get(), installation.getString(KEY_INSTALLATION_ID));
+    // Make sure we update the locale identifier
+    assertEquals("en-US", installation.getString(KEY_LOCALE_IDENTIFIER));
```

Comment:
Let's specify a locale with `Locale.setDefaultLocale(Locale)` since we might not always be running on a `en-US` device.